src/common/errors/ErrorMessages.ts

```diff
@@ -0,0 +1,5 @@
+// Unified location for error messages
+// TODO: move all errors here.
+
+export const ExploreSourceModelDoesntExist =
+  "Selected source model for the active metrics cube no longer exists";
```

Comment:
It was decided that for now we will just use the terminology "metrics" instead of "metrics cube" (or the other options). We might change this in the future, but Mike wanted to go with that for

Reply:
If we want to unify our error messages, I wonder if we would even want to use a string enum to contain them (for typing)? I'm not sure that is a good idea, just something that occurred to me

Reply:
Hmmm i was thinking of having functions in certain cases that handles messages that depend on variables. So depending on the message it will either be a constant or a funciton.

Reply:
oh i see, that makes sense